mrbelvedereci/plan/management/commands/compressplans.py

```diff
@@ -0,0 +1,96 @@
+from itertools import combinations
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+from mrbelvedereci.plan.models import Plan
+from mrbelvedereci.plan.models import PlanRepository
+from mrbelvedereci.utils import is_attr_equal
+
+
+class Command(BaseCommand):
+    help = '"Compress" plans by merging redundant plans into a single plan'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--delete',
+            action='store_true',
+            dest='delete',
+            default=False,
+            help='Delete plan instead of deactivating',
+        )
+        parser.add_argument(
+            '--dry-run',
+            action='store_true',
+            dest='dry_run',
+            default=False,
+            help='Do a dry run (do not modify db)',
+        )
+
+    def handle(self, *args, **options):
+        if options['dry_run']:
+            self.stdout.write('DRY RUN - db will not be modified')
```

Comment:
can we get a logger in this context?

Reply:
It seems we can get a logger but for some reason it is unable to print. This syntax is recommended in the guide here: https://docs.djangoproject.com/en/1.10/howto/custom-management-commands/